server/client.go

```diff
@@ -2206,7 +2218,10 @@ func (c *client) addSubToRouteTargets(sub *subscription) {
 }
 
 // This processes the sublist results for a given message.
-func (c *client) processMsgResults(acc *Account, r *SublistResult, msg, subject, reply []byte, queues *[][]byte) {
+// If the incoming message was for a service reply (subject starts with `_R_.`)
```

Comment:
This feels like it could use some rework eventually.

Reply:
Reverted all changes to this function.